quickstart.md

```diff
@@ -2,39 +2,20 @@
 
 # Perspective API Quickstart
 
-1.  **Get whitelisted.** To use the API, your you need to have your email
-    address whitelisted. [Apply here](https://www.perspectiveapi.com/). You'll
-    hear from someone on our team once you're approved.
+1.  **Create a Google Cloud project.** Go to the [Google Cloud console](https://console.developers.google.com/) to create a new project. You can also use an existing project, if you have one.
 
-2.  **Create a Google Cloud project.** Go to the [Google Cloud
-    console](https://console.developers.google.com/) to create a new project.
-    You can also use an existing project, if you have one.
+    *Note*: Your application doesn't need to be *hosted* on Google Cloud, but you will still need a Google Cloud project to authenticate your API requests.
 
-    *Note*: Your application doesn't need to be *hosted* on Google Cloud, but
-    you will still need a Google Cloud project to authenticate your API
-    requests.
-
-3.  **Enable the API.** Once you've been whitelisted, you'll be able to enable
-    the API in for your project. Go to the [Perspective API's overview
-    page](https://console.developers.google.com/apis/api/commentanalyzer.googleapis.com/overview)
-    and click **_Enable_**. If you haven't yet been whitelisted, you'll get a
-    permissions error at this point.
+2.  **Enable the API.** You'll be able to enable the API in for your project. 
```

Comment:
"You'll be able to enable the API" doesn't quite make sense anymore, since there's no whitelist step.

I think we can just delete this first sentence altogether?